src/pages/Home/CustomizeBobaModalBody.tsx

```diff
@@ -6,7 +6,30 @@ import { Container } from 'react-bootstrap';
 import { milkList } from 'assets/sampleBobaAPI';
 import CustomizeRadioButton from 'components/RadioButton';
 
-const BobaModalform = ({ milk, setMilk }: { milk: string; setMilk: (milk: string) => void }) => {
+interface BobaModalProps {
+  milk: string;
+  // toppings: {[key: string]:boolean};
+  setMilk: (milk: string) => void;
+  // setToppings: (toppings: {[key: string]:boolean}) => void;
+}
+
+const BobaModalform = ({ milk, setMilk }: BobaModalProps) => {
+  const radioBoxHandler = (option: string) => {
+    setMilk(option);
```

Comment:
if there's nothing more than just `setMilk` just pass this directly into `handlechange`

Reply:
but isnt it better to be more consistance since checkbox handler going to be more than 1 line and wouldnt it better to just have two separate handler than 1 with helper function and 1 without ?

Reply:
naming would be consistent but
You have to jump twice from `handleChange` to `handleRadioButton` to `setMilkType`
while using `setMilkType` knows we just jump straight to `useState` to check.
BigO also saves space coz we don't need to assign variable and saving memory and declaring `option: string` for props

Reply:
@seintun